docs/tutorial.md

```diff
@@ -257,3 +257,27 @@ sensitive_unshuffled_avatars_df = pd.read_csv(
 )
 print(avatars_df.head())
 ```
+
+## Reset your password
+
+**NB**: This section is only available if the use of emails to login is activated in the global configuration. It is not the case by default.
+
+If you forgot your password or it you need to set one, first notify that you forgot it:
+
+```python
+from avatars.client import apiclient
+
+client = apiclient(base_url=os.environ.get("BASE_URL"))
+client.forgotten_password("yourmail@mail.com")
```

Comment:
```suggestion
from avatars.client import ApiClient

client = ApiClient(base_url=os.environ.get("BASE_URL"))
client.forgotten_password("yourmail@mail.com")
```

Reply:
Also, don't they need to use `ForgottenPasswordRequest` ? 
Same with `ResetPasswordRequest` ?

Reply:
Nope, I did not update the code yet but it's not necessary :)